### PR TITLE
Bug 1221924 - [TV][usage] Log app usage time and smart-home covering time separately

### PR DESCRIPTION
**Pull request by rexboy7**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c165e2c981fb86f08640c38a805229b0daeff32d

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=09065497d6416cce17233322cdb11720d209a2e5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=09065497d6416cce17233322cdb11720d209a2e5

File: tv_apps/smart-system/js/app_usage_metrics.js [hunk 1]
```diff
@@ -430,17 +432,42 @@
...
     case APPOPENED:
-    case HOMESCREEN:
-      // The user has opened an app, switched apps, or switched to the
-      // homescreen. Record data about the app that was running and then
+      // The user has opened an app or switched apps.
+      // Record data about the app that was running and then
       // update the currently running app.
-      this.metrics.recordInvocation(this.getCurrentApp(),
-                                    now - this.getCurrentStartTime());
+      if (this._appIsUnderlay) {
```
Comment (luke-chang) [inline tv_apps/smart-system/js/app_usage_metrics.js]: Please add some comments to describe when this condition will happen (as we discussed).

File: tv_apps/smart-system/js/app_usage_metrics.js [hunk 2]
```diff
@@ -886,6 +913,24 @@
     return time > 0;
   };
 
+  UsageData.prototype.recordUsageTime = function(app, time) {
```
Comment (luke-chang) [inline tv_apps/smart-system/js/app_usage_metrics.js]: Please add comments to explain this method's purpose.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6b9d65826d72fcc5aefbd25714f383f579c02d87

**Pull request merged**